src/implementation/bn.ts

```diff
@@ -0,0 +1,60 @@
+import { throwNotImplemented } from "..";
```

Comment:
Is this file used for?

Reply:
This file just implements an EmptyBigNumber class. The real implementation is in the plugin instance.